google-chart-loader.js

```diff
@@ -0,0 +1,385 @@
+import '@polymer/polymer/polymer-legacy.js';
```

Comment:
It's hard to see the changes to this file

Reply:
The only manual and meaningful changes are: `initializeOnce` and `_loadPackagesDebounce`.
The rest was not changed. This was originally `google-chart-loader.html`.

Reply:
Gotcha.